### PR TITLE
Fixed #21113 -- Made LogEntry.change_message language independent

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by RamezIssac** [comment]
:+1:

**Comment by claudep** [comment]
Tim, I think the patch is ready for a second round of review.

File: django/contrib/admin/models.py [hunk 1]
```diff
@@ -85,6 +91,46 @@ def is_change(self):
     def is_deletion(self):
         return self.action_flag == DELETION
 
+    def get_changes(self):
```
Comment (timgraham) [inline django/contrib/admin/models.py]: What do you think about naming this something like `get_change_message()`. I feel like `get_changes()` implies a return type of something like list.

**Comment by timgraham** [comment]
The `object_history.html` template is still accessing `{{ action.change_message }}` directly. ;-)

**Pull request merged**